code/game/area/Space Station 13 areas.dm

```diff
@@ -170,7 +170,13 @@ proc/process_adminbus_teleport_locs()
 /area/no_ethereal
 	anti_ethereal = 1
 
-
+/area/dojo
+	name = "\improper Spider Clan Dojo"
+	icon_state = "dojo"
+	requires_power = 0
```

Comment:
Might want an area where they can practice draining APCs

Reply:
definitely among the things i want to add, but not yet